song/routers/likeRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+
+const router = express.Router();
+
+const likeController = require("../controllers/likeController");
+
+router.post("/like", likeController.like);
```

Comment:
```suggestion
router.post("", likeController.like);
```
이미 routes의 index.js를 통해 `/likes`로 한 번 분기되었기 때문에 추가로 엔드포인트가 likeRouter.js에서도 작성될 필요는 없을 것 같습니다. 
```shell
$ http -v POST 127.0.0.1:3000/likes/like     # 현재 작성하신 엔드포인트

$ http -v POST 127.0.0.1:3000/likes          # 수정된 엔드포인트

```